tests/tests.rs

```diff
@@ -467,7 +467,11 @@ fn test_sse42_chunked_panic() {
 // FIXME: remove this comment once this works properly.
 #[test]
 #[should_panic]
-#[cfg(all(feature = "public_imp", target_arch = "wasm32"))]
+#[cfg(all(
+    feature = "public_imp",
+    target_arch = "wasm32",
+    target_feature = "simd128"
+))]
```

Comment:
The CI integration exposed this bug--the test should only be exposed if WASM SIMD is enabled.